pandas/core/frame.py

```diff
@@ -5592,22 +5592,79 @@ def corrwith(self, other, axis=0, drop=False):
 
     def count(self, axis=0, level=None, numeric_only=False):
         """
-        Return Series with number of non-NA/null observations over requested
-        axis. Works with non-floating point data as well (detects NaN and None)
+        Count non-NA cells for each column or row.
+
+        Return Series with number of non-NA observations over requested
```

Comment:
One last change, maybe remove the first sentence since this can return a DataFrame with `level`.

I think just use the extended summary to say what counts as non-null data.

The values `None`, `NaN`, `NaT`, and optionally `np.inf` (depending on ``pandas.options.mode.use_inf_as_na``) are considered NA.

Reply:
Do you mean the first sentence in the extended summary, i.e. :
"Return Series with number of non-NA observations over requested axis."

If I understand you right I would change the entire summary (i.e. short and extended summary) to look like the following:
```
        Count non-NA cells for each column or row.

        The values None, NaN, NaT, and optionally np.inf (depending on
        pandas.options.mode.use_inf_as_na) are considered NA.
```

Reply:
Yeah. np.inf to \`numpy.inf\` and single backticks around pandas.options.mode.use_inf_as_na.